xml/System.ServiceModel.Description/DataContractSerializerOperationBehavior.xml

```diff
@@ -471,8 +472,8 @@
         <ReturnType>System.Runtime.Serialization.ISerializationSurrogateProvider</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Gets or sets a surrogate provider to be used during the serialization and deserialization processes.</summary>
+        <value>An <see cref="T:System.Runtime.Serialization.ISerializationSurrogateProvider" /> that is used to provide a serialization surrogate used during serialization and deserialization.</value>
```

Comment:
Given that the type is automatically shown (https://review.docs.microsoft.com/en-us/dotnet/api/System.ServiceModel.Description.DataContractSerializerOperationBehavior.SerializationSurrogateProvider?branch=pr-en-us-3748&view=dotnet-plat-ext-3.1#property-value), I think you can have the plain text here too

```suggestion
        <value>A surrogate provider that is used to provide a serialization surrogate to be used during serialization and deserialization.</value>
```